includes/wp-admin/class-settings-fields.php

```diff
@@ -75,6 +75,14 @@ public static function register_settings_fields() {
 			'activitypub_activities'
 		);
 
+		add_settings_field(
```

Comment:
let's also check for `ACTIVITYPUB_DISABLE_INCOMING_INTERACTIONS`, to show or hide the settings.